src/app/portlets/dot-templates/dot-template/dot-template.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="content-wrapper">
+    <form class="p-fluid" [formGroup]="group" (ngSubmit)="onSubmit($event)">
+        <button pButton label="Save" [disabled]="isDisabled" class="p-button-secondary"></button>
+        <div class="p-field">
+            <label for="title">Title</label>
+            <input id="title" formControlName="title" type="text" placeholder="Title" pInputText />
```

Comment:
https://github.com/dotCMS/core-web/pull/1497/files#r517397880 in `Placeholder`

Reply:
On hold for now.